lib/active_admin/views/index_as_table.rb

```diff
@@ -213,7 +213,7 @@ def build(page_presenter, collection)
         table_options = {
           id: "index_table_#{active_admin_config.resource_name.plural}",
           sortable: true,
-          class: "index_table index",
+          class: "index-table js-index-table",
```

Comment:
In what ways could JS break the styles applied by CSS? I don't understand the benefit of having a separate class.